src/components/DpEditableList/DpEditableList.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <div>
     <ul
-      class="u-mb-0_75"
-      v-if="entries.length > 0 || Object.keys(entries).length > 0">
+      v-if="entries.length > 0 || Object.keys(entries).length > 0"
```

Comment:
shoudn't be thus sufficient?
```suggestion
      v-if="Object.keys(entries).length > 0"
```